### PR TITLE
Add hyperblue theme

### PR DESCRIPTION
**Pull request by tylerreckart**
Updated the README to include hyperblue theme

## Checklist:

- [x] The title for my package or theme uses its npm title (hyperterm-plugin-example)
- [x] The link for my package or theme uses the npmjs.com link (https://npmjs.com/package/hyperterm-plugin-example)
- [x] There is a visual representation of what my plugin does in the repo. (Plugin: screenshot or gif || Theme: screenshot of the colors within HyperTerm)
- [x] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme of why it's awesome and deserves to be on the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Like `awesome-hyperterm`? Reach out to me and say *hi* on [Twitter](https://twit
 
 # Themes
 * [hyperterm-atom-dark](https://www.npmjs.com/package/hyperterm-atom-dark) - Dark - Really beautiful import of Atom One Dark theme from the [official Atom theme](https://github.com/atom/one-dark-syntax).
+* [hyperblue](http://npmjs.com/package/hyperblue) – Dark, cool hues. Turns HyperTerm blue.
```
Comment (bnb) [inline README.md]: Can you change it to `https://npmjs.com` please? HTTP throws a redirect. Want to keep the entire list on the HTTPS for the sake of consistency.

**Comment by tylerreckart** [comment]
@bnb Updated the pr as requested 😄

**Comment by bnb** [comment]
@tylerreckart \*sigh*.

This is my bad (didn't notice), but could you add `www` to npm as well? They automatically redirect `npmjs.com` > `www.npmjs.com`, which is really annoying.

**Comment by tylerreckart** [comment]
@bnb No problem. Updated

**Comment by tylerreckart** [comment]
@bnb Here's the updated PR that includes Hyperzsh in the resources section. Cheers!

**Comment by bnb** [comment]
Merged! Thanks for the contribution!

**Pull request merged**